Sources/SwiftProtobufCore/BinaryEncoder.swift

```diff
@@ -13,9 +13,8 @@
 ///
 // -----------------------------------------------------------------------------
 
-import Foundation
-
 /// Encoder for Binary Protocol Buffer format
+@usableFromInline
```

Comment:
`@inlinable` and `@usableFromInline` have no API impact but do have ABI impact. What happens with `@inlinable` methods is that their whole body is put into the swift interface files so that dependent modules can specialise their code when calling them.

Reply:
Sorry, right ABI vs. API, but sorta tied, in that my concern is without this, we could change these things in point release without any concerns, but someone has protos in a different framework, then it would seem like we're more locked in, no?